pkg/controller/vpcpeering/peering.go

```diff
@@ -136,11 +136,18 @@ func (e *external) Create(ctx context.Context, mg resource.Managed) (managed.Ext
 
 	peer.Status.SetConditions(xpv1.Creating())
 
-	_, err := e.compute.Networks.AddPeering(peer.Spec.ForProvider.Project, peer.Spec.ForProvider.Network, &compute.NetworksAddPeeringRequest{
-		Name:             peer.Spec.ForProvider.Name,
-		PeerNetwork:      peer.Spec.ForProvider.PeerNetwork,
-		AutoCreateRoutes: true,
```

Comment:
Why remove this field? Does the other option imply this field?

Reply:
they are replaced by sub fields in the `NetworkPeering`, as the comment said:
```go
	// NetworkPeering: Network peering parameters. In order to specify route
	// policies for peering using import and export custom routes, you must
	// specify all peering related parameters (name, peer network,
	// exchange_subnet_routes) in the network_peering field. The
	// corresponding fields in NetworksAddPeeringRequest will be deprecated
	// soon.
	NetworkPeering *NetworkPeering `json:"networkPeering,omitempty"`
```

And If set these fields, the result of request is invalid request after I tried.